frameworks/Java/play2-java/play2-java-ebean-hikaricp-netty.dockerfile

```diff
@@ -1,5 +1,8 @@
-FROM hseeberger/scala-sbt:8u151-2.12.5-1.1.2
+FROM hseeberger/scala-sbt:8u171_2.12.6_1.1.5
 WORKDIR /play2
 COPY play2-java-ebean-hikaricp .
+
+RUN sed -i 's/.enablePlugins(PlayJava, PlayEbean, PlayNettyServer)/.enablePlugins(PlayJava, PlayEbean, PlayNettyServer).disablePlugins(PlayAkkaHttpServer)/g' build.sbt
+
 RUN sbt stage
-CMD ["target/universal/stage/bin/play2-java-ebean-hikaricp", "-Dplay.server.provider=play.core.server.NettyServerProvider"]
+CMD ["target/universal/stage/bin/play2-java-ebean-hikaricp", "-Dplay.server.provider=play.core.server.NettyServerProvider", "-J-server", "-J-Xms1g", "-J-Xmx1g", "-J-XX:NewSize=512m", "-J-XX:+UseG1GC", "-J-XX:MaxGCPauseMillis=30", "-J-XX:-UseBiasedLocking", "-J-XX:+AlwaysPreTouch"]
```

Comment:
I found this being used on 2 of the scala versions and nowhere else. So I added it everywhere. For comparison's sake, all configurations for the same framework should use the same parameters as much as possible.